protobufs/livekit_room.proto

```diff
@@ -61,10 +61,9 @@ service RoomService {
   // Update room metadata, will cause updates to be broadcasted to everyone in the room, Requires `roomAdmin`
   rpc UpdateRoomMetadata (UpdateRoomMetadataRequest) returns (Room);
 
-  // Forward a participant's track to another room. Requires `roomAdmin` on the room whose name 
-  // consists of the source room and destination room as `source|dest`. The forwarding will stop 
```

Comment:
Do you want to mention that perms are needed with new field `ForwardDestRoom` filled in?